google-cloud-spanner-hibernate-dialect/src/test/java/com/google/cloud/spanner/hibernate/GeneratedUpdateTableStatementsTests.java

```diff
@@ -94,16 +94,15 @@ public void testUpdateStatements_alterTables() throws SQLException {
     List<String> sqlStrings = mockConnection.getStatementResultSetHandler().getExecutedStatements();
     assertThat(sqlStrings).containsExactly(
         "START BATCH DDL",
-        "alter table Employee ADD COLUMN id INT64 not null",
```

Comment:
Hibernate 5.5.2.final is resulting in some schema changes when updating a table that already exists.

Reply:
Would you describe the problem a bit more?

These deleted lines look as if hibernate 5.6.5.Final doesn't work with these alter table statements.

Reply:
Experimented with this a bit. Versions 5.4.6 and 5.5.0.final resulted in no change in schema, however versions 5.5.2.final onwards appear to drop the command that adds columns to a pre-existing table. This looks like a schema update from hibernate orm. Optimizations tend to be put in place with every [version upgrade ](https://in.relation.to/2021/06/15/hibernate-orm-552-release/) so I wonder if this is related. Also found a similar issue that is related to updating existing tables: https://github.com/hibernate/hibernate-orm/pull/3151

Reply:
I see. Would you add source code comment `// Why not seeing alter table here? It's because in hibernate 5.5.2 onwards ...`

Reply:
Good idea! It'll be helpful to mention this.